ontology/yaml/resources/HVAC/entity_types/HWS.yaml

```diff
@@ -229,3 +229,24 @@ HWS_US_MTV_2000_2:
   uses:
   - mixing_valve_percentage_command
   - heat_exchange_supply_water_isolation_valve_command
+
+
+# Non Standard Types for CH ZRH EURD
+HWS_SWTC_CPC:
+  description: "Simple hot water system with circulation pump"
+  implements:
+  - HWS
+  - SWTC
+  - CPC
+  uses:
+  - circulation_pump_speed_percentage_command
+  - circulation_pump_speed_percentage_setpoint
+  - failed_alarm
+
+HWS_NON_CANONICAL_TYPE_1:
```

Comment:
We dont need to call these out as incomplete here, you would translate them incompletely. This thing would simply be HWS_SWTC and would be canonical; the specific entity in building config would implement HWS_SWTC and have missing fields if they weren't there.